travel_platform/api/skyscanner/skyscanner_data_tests.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from skyscanner_data import SkyscannerData
+
+# FIXME: Sasha: Fix all these tests, they fails because of invalid parameters.
```

Comment:
Are you sure that it's currently not working?
I've fixed something in your task few days ago.